definitions/npm/react-image-crop_v3.x.x/flow_all/react-image-crop_v3.x.x.js

```diff
@@ -0,0 +1,50 @@
+declare module 'react-image-crop' {
+
+	declare type PercentCrop = {
+		x?: number,
+		y?: number,
+		width?: number,
+		height?: number
+	};
+
+	declare type PixelCrop = {
```

Comment:
`PixelCrop` is the output of some of the internals and callbacks, and it's always in this shape. But I also use it in the type `AspectCrop`, because that also has the shape of `PixelCrop` but it has an optional `aspect` key.

`PercentCrop` has no restriction set by the code. Everything that uses it always checks if a key is in the object. I was not sure how to better write down this 3 objects, that is why they're 3 separate types, but I think they're descriptive enough. Suggestions are welcome.